### PR TITLE
Actions two stage

### LINKED ISSUE #686: make application actions two-stage
- user should click on "Approve..." then a text box opens underneath, then finally a click on "Approve" makes the action
- also make the prompts & labels specific to the action, e.g. "Comment for approval"

### PR DESCRIPTION
**Pull request by Koura**
Closes #686

### CODE REVIEW

File: src/cljs/rems/application.cljs [hunk 1]
```diff
@@ -719,38 +719,126 @@
...
+  [action-tab "withdraw" (text :t.actions/withdraw)])
+
+(defn- review-request-tab []
+  [action-tab "review-request" (text :t.actions/review-request)])
+
+(defn- action-comment [label-title]
+  [:div.form-group
+   [:label {:for "judge-comment"} label-title]
+   [textarea {:id "judge-comment"
+              :name "judge-comment" :placeholder "Comment"
```
Comment (luontola) [inline src/cljs/rems/application.cljs]: Translation missing.

File: src/cljs/rems/application.cljs [hunk 2]
```diff
@@ -719,38 +719,126 @@
    {:type "button" :data-toggle "modal" :data-target "#review-request-modal"}
    (text :t.actions/review-request)])
 
+;;; Actions tabs
+
+(defn- action-tab [action-name text]
+  (let [action-name (str "actions-" action-name)]
+    [:a.nav-item.nav-link
+     {:id (str action-name "-tab") :data-toggle "tab" :href (str "#" action-name) :role "tab" :aria-controls action-name :aria-selected false}
```
Comment (luontola) [inline src/cljs/rems/application.cljs]: Very long line. Wrap to multiple lines?

Comment (Koura) [reply src/cljs/rems/application.cljs]: It's a bit unwieldy, I'll fix it.

File: src/cljs/rems/application.cljs (hunk 1 above)
Comment (Koura) [reply src/cljs/rems/application.cljs]: Good catch, it's properly been that way for a long time.

**Pull request merged**